include/spectrum.h

```diff
@@ -334,6 +334,10 @@ extern uint  __LIB__    zx_screenstr_callee(uchar row, uchar col) __smallc __z88
 #define zx_setcursorpos(a,b)      fputc_cons(22); fputc_cons(a); fputc_cons(b);
 #define zx_topleft()              fputc_cons(22); fputc_cons(0); fputc_cons(0);
 
+// Print a string on the screen
+extern int printk(char *fmt, ...);
+#define zx_print(s) printk(s)
```

Comment:
This won't work for the case `zx_print("%d %d",12,12);` for example

Reply:
An alternative alias would be to puts_cons(x) which takes a single string and just writes it out to the console. It's a lot smaller than printk as well.